### ISSUE TITLE
Fix hot or cold reloading

### ISSUE THREAD

**Issue by Odilf** [Odilf/Odilf-Timer#28]

**Comment by Odilf** [comment]
I hate my life. I spent HOURS trying to fix this, going through all the commit history until I made a brand new project with the template and moved everything script by script. Turns out I move everything and it still worked. What was the problem? I was editing properties in a sass `<style>` tag. I reckon because of the `include_paths['theme']` in the svelte preprocessor, it's getting confused and not picking up changes inside in said scss style tags.

**Comment by Odilf** [comment]
Ok so I've spent another half hour trying to fix this because the style tags weren't the problem. My conclusion is that the problem is that I renamed the component for `footer.svelte` to `Footer.svelte` (capitalization). Then, git in vscode or something made it so that deleting the file and creating it again didn't fix it. I don't know if the problem lies in SvelteKit, Vite, Git or VsCode at this point. 
I fixed (I think (I hope...)) it by commiting the deletion of `Footer.svelte` and then creating a new one.
I pray to God it's actually working now. 

Sidenote: scss had nothing to do with it, as far as I can tell. Still, i wouldn't discard the possibility of being the path thing I mentioned before.
